pkgs/os-specific/linux/kernel/common-config.nix

```diff
@@ -431,7 +431,7 @@ let
       F2FS_FS_COMPRESSION = whenAtLeast "5.6" yes;
       UDF_FS              = module;
 
-      NFSD_V2_ACL            = yes;
+      NFSD_V2_ACL            = whenOlder "6.2" yes;
```

Comment:
Just noting for the record (in case somebody comes searching) that this option still exists, but NFSD_V2 now needs to be manually enabled.  The description is:

> NFSv2 (RFC 1094) was the first publicly-released version of NFS.  Unless you are hosting ancient (1990's era) NFS clients, you don't need this.

So letting that go away with 6.2 sounds fine to me.